bin/action_subscriber

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+
+require 'active_support'
+require 'active_support/core_ext'
+require 'thor'
+
+module ActionSubscriber
+  class CLI < ::Thor
+    class_option :app, :default => "./config/environment.rb"
+    class_option :mode
+    class_option :host
+    class_option :hosts
+    class_option :pop_interval, :type => :numeric, :desc => "how long to wait between asking for messages (in milliseconds)"
+    class_option :port, :type => :numeric
+    class_option :threadpool_size, :type => :numeric
+    class_option :times_to_pop, :type => :numeric, :desc => "how many messages to get from each queue each time we ask rabbit"
+
+    desc "start", "Start the babou subscription server"
+    long_desc <<-BABOUDESC.strip_heredoc
+      Babou is a simple subscriber server to manage event subscriptions in a separate process.
+    BABOUDESC
+
+    def start
+      require options[:app]
+      require 'action_subscriber/babou'
+
+      $0 = "babou server #{object_id}"
+      puts "Desperate for something to play with..."
+
+      ::ActionSubscriber::Babou.configure_from_yaml_and_cli(options)
+      case ::ActionSubscriber.configuration.mode
+      when /pop/i then
+        ::ActionSubscriber::Babou.auto_pop!
+      when /subscribe/i then
+        ::ActionSubscriber::Babou.start_subscribers
+      end
+    end
+  end
+
+  [:INT, :QUIT, :TERM].each do |signal|
+    trap(signal) do
+      require 'babou'
+
+      puts "Down Babou! ... down!"
+      puts "ROAAAAAARRRRRRRR!!!!!!"
+      wait_loops = 0
+      ::Babou.stop_server!
+
+      # Going to wait until the thread pool drains or we wait for 1000 seconds
+      # Only waiting for shut down in pounce mode
+      if ::Babou.pounce?
```

Comment:
`Babou` is no longer a top-level module, should be `ActionSubscriber::Babou`, also we should change this `pounce` to `pop`